---
title: "Impact of Antibiotic Treatment on the Gut Microbiome and Its Resistome in Hematopoietic Stem Cell Transplant Recipients"
draft: false
authors:
- "N{\o}rgaard, Jens Christian"
- "J{\o}rgensen, Mette"
- "Moestrup, Kasper Sommerlund"
- "Ilett, Emma Elizabeth"
- "Zucco, Adrian Gabriel"
- "Marandi, Ramtin Z"
- "Julian, Marc Noguera"
- "Paredes, Roger"
- "Lundgren, Jens D"
- "Sengel{\o}v, Henrik"
- "MacPherson, Cameron"
date: "2023-07-01"
doi: "10.1093/infdis/jiad033"
category: ["article-journal"]
journal: "The Journal of Infectious Diseases"
summary: "Antibiotic-resistant bacterial infections are increasingly an issue in allogenic hematopoietic stem cell transplant patients. How antibiotic treatment impacts antibiotic resistance in the human gut microbiome remains poorly understood in vivo. Here, a total of 577 fecal samples from 233 heavily antibiotic-treated transplant patients were examined using high-resolution prescription data and shotgun metagenomics. The 13 most frequently used antibiotics were significantly associated with 154 (40% of tested associations) microbiome features. Use of broad-spectrum $\beta$-lactam antibiotics was most markedly associated with microbial disruption and increase in resistome features. The enterococcal vanA gene was positively associated with 8 of the 13 antibiotics, and in particular piperacillin/tazobactam and vancomycin. Here, we highlight the need for a high-resolution approach in understanding the development of antibiotic resistance in the gut microbiome. Our findings can be used to inform antibiotic stewardship and combat the increasing threat of antibiotic resistance."
links:
- name: "DOI"
  url: "https://doi.org/10.1093/infdis/jiad033"
featured: false
---

## Abstract

Antibiotic-resistant bacterial infections are increasingly an issue in allogenic hematopoietic stem cell transplant patients. How antibiotic treatment impacts antibiotic resistance in the human gut microbiome remains poorly understood in vivo. Here, a total of 577 fecal samples from 233 heavily antibiotic-treated transplant patients were examined using high-resolution prescription data and shotgun metagenomics. The 13 most frequently used antibiotics were significantly associated with 154 (40% of tested associations) microbiome features. Use of broad-spectrum $\beta$-lactam antibiotics was most markedly associated with microbial disruption and increase in resistome features. The enterococcal vanA gene was positively associated with 8 of the 13 antibiotics, and in particular piperacillin/tazobactam and vancomycin. Here, we highlight the need for a high-resolution approach in understanding the development of antibiotic resistance in the gut microbiome. Our findings can be used to inform antibiotic stewardship and combat the increasing threat of antibiotic resistance.
